Sources/BezierSwift/Foundation/Color/Token/FunctionalColorToken.swift

```diff
@@ -0,0 +1,374 @@
+//
+//  FunctionalColorToken.swift
+//
+//
+//  Created by Tom on 4/24/24.
+//
+
+import UIKit
+import SwiftUI
+
+internal struct FunctionalColorToken: BezierColorType {
+  let lightColorToken: ColorTokenType
+  let darkColorToken: ColorTokenType
+  let pressedColorToken: ThemeableColorTokenType
+  
+  init(
+    lightGlobalColorToken: GlobalColorToken,
+    darkGlobalColorToken: GlobalColorToken,
+    pressedColorToken: ThemeableColorTokenType
+  ) {
+    self.lightColorToken = lightGlobalColorToken
+    self.darkColorToken = darkGlobalColorToken
+    self.pressedColorToken = pressedColorToken
+  }
+}
+
+// MARK: - ForegroundBlue
+extension FunctionalColorToken {
+  static var fgBlueNormal = FunctionalColorToken(lightGlobalColorToken: .blue400, darkGlobalColorToken: .blue300, pressedColorToken: PressedColorToken.fgBlueNormal)
```

Comment:
```suggestion
  static let fgBlueNormal = FunctionalColorToken(lightGlobalColorToken: .blue400, darkGlobalColorToken: .blue300, pressedColorToken: PressedColorToken.fgBlueNormal)
```
변경되지 않는 값이라면 전체적으로 let 으로 처리되면 좋을 것 같아요